include/z64.h

```diff
@@ -1764,20 +1764,27 @@ typedef struct {
     /* 0x10 */ s16 unk_10;
 } JpegDecoderState; // size = 0x14
 
+typedef enum {
+    VI_MODE_EDIT_STATE_INACTIVE,
+    VI_MODE_EDIT_STATE_ACTIVE,
+    VI_MODE_EDIT_STATE_2, // active, more adjustments
+    VI_MODE_EDIT_STATE_3  // active, more adjustments, print comparison with NTSC LAN1 mode
+} ViModeEditState;
```

Comment:
```suggestion
typedef enum {
    /* 0 */ VI_MODE_EDIT_STATE_INACTIVE,
    /* 1 */ VI_MODE_EDIT_STATE_ACTIVE,
    /* 2 */ VI_MODE_EDIT_STATE_2, // active, more adjustments
    /* 3 */ VI_MODE_EDIT_STATE_3  // active, more adjustments, print comparison with NTSC LAN1 mode
} ViModeEditState;
```
Just a small cleanup suggestion